roles/nuage_master/templates/nuage-strongswan-pod-config-daemonset.j2

```diff
@@ -0,0 +1,45 @@
+# This manifest installs StrongSwan on
+# each worker node on a EVDF OSE cluster.
+kind: DaemonSet
+apiVersion: extensions/v1beta1
+metadata:
+  name: nuage-strongswan-ds
+  namespace: kube-system
+  labels:
+    k8s-app: nuage-strongswan-ds
+spec:
+  selector:
+    matchLabels:
+      k8s-app: nuage-strongswan-ds
+  updateStrategy:
+    type: RollingUpdate
+  template:
+    metadata:
+      labels:
+        k8s-app: nuage-strongswan-ds
+    spec:
+      hostNetwork: true
+      tolerations:
+        - key: node-role.kubernetes.io/master
+          effect: NoSchedule
+          operator: Exists
+      containers:
+        # This container installs strongswan running as a
+        # container on each worker node
+        - name: install-strongswan
+          image: nuage/strongswan:{{ nuage_strongswan_container_image_version }}
```

Comment:
It would be nice to set the image name in `defaults.yml` of the role, so that it can be overriden